.github/dependabot.yml

```diff
@@ -0,0 +1,22 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/chess"
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "npm"
+    directory: "/gila/frontend"
```

Comment:
i'm assuming this is how gila needs to be structured due to the fronted/backend structure

Reply:
@dtbuchholz This looks right to me, thanks for setting this up